src/opm/parser/eclipse/EclipseState/Aquifer/Aquancon.cpp

```diff
@@ -108,6 +108,7 @@ namespace Opm {
 
     Aquancon::Aquancon(const EclipseGrid& grid, const Deck& deck)
     {
+        const std::vector<int>& actnum = grid.getACTNUM();
```

Comment:
From here, the using of pinched grid for connection generation, will not affect the analytical aquifers. This PR is for numerical aquifers, the analytical aquifers will be handled in a separate PRs for the ease of testing and analysis.